### ISSUE TITLE
Improve Speed Feedback configuration experience (first startup)

### ISSUE THREAD

**Issue by ted-miller** [Yaskawa-Global/motoros2#45]
The first startup, it connected to the agent. Then it took a LONG time to init each control group. During that initialization, it disconnected from the agent. The listening script didn't indicate why it disconnected.

Eventually I got the expected alarm. But it took a while to init all four groups and I was on the verge of panic.

This should be improved. (or made unncessary).

The excessive delay was noticed on an `R1 S1 S2 S3` system.

> Would it perhaps already help if we postpone connecting to the Agent until after the Controller struct has been initialised (ie: all system level / non-micro-ROS stuff has been initialised)? 
> That way there's at least not the confusion caused by (half) connecting and then timing out of the Agent connection and the weird interplay between alarms and all the micro-ROS stuff going on.

Probably. But it would need some refactoring, as Controller init relies on the agent.

**Comment by gavanderhoorn** [comment]
@ted-miller: now that the installation procedure was updated (#100, disable FSU CRC before install), do you believe this is still an issue?

**Comment by ted-miller** [comment]
Yes, this is still an issue. The CRC check is a separate issue in my mind.
